doc/install_windows.txt

```diff
@@ -0,0 +1,608 @@
+
+   dlltool --dllname python27.dll --def python27.def --output-lib WinPython-64bit-2.7.6.4\python-2.7.6.amd64\libs\libpython27.a
+
+Installing Theano
+~~~~~~~~~~~~~~~~~
+
+Once the dependencies are installed, you can download and install
+Theano. We have found that in the long run, the Git install is the
+most useful, because you can update it with a single ``git pull``
+command. Therefore we recommend it. However, a manual install without
+Git is also possible.
+
+Git Install
+###########
+
+
+Theano is hosted on GitHub, you need Git to download it. For Windows,
+download and install the `MSYSGIT <http://msysgit.github.io/>`_ build.
+Open up the `Git Shell` in the directory in which you want to install
+Theano. For the bleeding-edge version execute
+
+.. code-block:: bash
+
+  git clone https://github.com/Theano/Theano.git
+
+For the latest stable release 0.6 (as of Oct 2014) run instead:
+
+.. code-block:: bash
+
+  git clone https://github.com/Theano/Theano.git --branch rel-0.6
+
+Either way, a folder `Theano` will be created with the library
+downloaded to it.
+
+
+Manual Installation
+###################
+
+To get the latest bleeding edge version got to `Theano on GitHub
+<https://github.com/Theano/Theano>`_ and download the `latest zip
+<https://github.com/Theano/Theano/archive/master.zip>`_. Then unzip it
+somewhere.
+
+Alternatively, you can check the latest release release 0.6 (as of Oct
+2014) by going to
+https://github.com/Theano/Theano/releases/tag/rel-0.6 and
+downloading the `zip
+<https://github.com/Theano/Theano/archive/rel-0.6.zip>`_.
+
+Configuring Theano
+##################
+
+Once you have installed Theano, open the Python Shell
+(e.g. ``c:\scisoft\env.bat`` if you follow the installation directories
+from this tutorial) and ``cd`` to it. Then run::
+
+  python setup.py develop
+
+this step will add the Theano directory to you ``PYTHON_PATH``
+environment variable.
+
+At this stage you can check whether Theano works and is able to
+compile C code for CPU execution.
+
+Create a test file containing:
+
+.. code-block:: python
+
+   import numpy as np
+   import time
+   import theano
+   A = np.random.rand(1000,10000).astype(theano.config.floatX)
+   B = np.random.rand(10000,1000).astype(theano.config.floatX)
+   np_start = time.time()
+   AB = A.dot(B)
+   np_end = time.time()
+   X,Y = theano.tensor.matrices('XY')
+   mf = theano.function([X,Y],X.dot(Y))
+   t_start = time.time()
+   tAB = mf(A,B)
+   t_end = time.time()
+   print "NP time: %f[s], theano time: %f[s] (times should be close when run on CPU!)" %(
+                                              np_end-np_start, t_end-t_start)
+   print "Result difference: %f" % (np.abs(AB-tAB).max(), )
+
+Then run it. It should execute without problems and the Theano function
+should run at a speed similar to the regular NumPy
+multiplication. (Both Numpy and Theano should call the same BLAS
+routine for matrix multiplication)
+
+Running Theano's test-suite
```

Comment:
I meant putting this section as a subsection of "Installing Theano" not "Configuring Theano" (essentially leave it in the same place in the file but underline it with ######### instead of +++++++ so that Sphynx will know that it's under "Installing Theano", not the section on using a gpu with Theano